examples/celery-rabbitmq/rabbitmq-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  labels:
+    name: rabbitmq-service
+  name: rabbitmq-service
+spec:
+  port: 5672
```

Comment:
What version of Kubernetes/kubectl are you using? v1beta3 was only complete as of release 0.15.0. There is no port field in the final spec:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L932